calicoctl/commands/node/diags.go

```diff
@@ -155,7 +160,43 @@ such as transfer.sh using curl or similar.  For example:
 
 }
 
-// writeDiags executes the dignostic commans and outputs the result in the file
+// getDockerLogs will attempt to grab logs for any "calico" named containers for hosted installs.
+func getDockerLogs(logDir string) {
+	os.Mkdir(logDir, os.ModeDir)
+
+	fmt.Println("Copying logs from Calico containers")
```

Comment:
we should move this after line 182, since if we don't find any Calico containers then we print this message suggesting we're collecting logs but in fact we are not (and the debug log goes silently unnoticed by the user)